.github/workflows/ci.yml

```diff
@@ -60,7 +60,8 @@ jobs:
           - target: Linux
             state: NU7
             os: ubuntu-latest
-            rustflags: '--cfg zcash_unstable="nu7"'
+            # TODO: Enable NU7 tests when zebra-chain used revision supports it.
+            #rustflags: '--cfg zcash_unstable="nu7"'
```

Comment:
This change I disagree with. It is entirely expected that these test cases be allowed to fail and the PR can still be merged; that is why the `required-checks` job doesn't check this one. It would be no different if macOS or Windows builds started failing; we would permit PRs that break Zallet on those platforms to be merged because they are not (currently) target platforms.

The current test failure is actually a Zebra bug: Zebra should be using `#[cfg(zcash_unstable="nu7"]` annotations on NU7-specific code instead of commenting them out.

Reply:
Makes sense, thanks. Deferred the feature issue to Zebra. https://github.com/ZcashFoundation/zebra/issues/9811